<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Storemanager</class>
 <widget class="QMainWindow" name="Storemanager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bulk Club</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/imgs/imgs/icon.png</normaloff>:/imgs/imgs/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1601</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(202, 210, 197);
font:  700 40pt &quot;Nirmala UI&quot;;
background-color: rgb(53, 79, 82);</string>
    </property>
    <property name="text">
     <string>               Store Managment System</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/imgs/imgs/store.png);
border-radius:50%;</string>
    </property>
   </widget>
   <widget class="QLabel" name="DateTime">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>0</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 79, 82);
font:25  &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>mm/dd/yyyy hh:mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(132, 169, 140);
font: 14pt &quot;Nirmala UI&quot;;
color: rgb(255, 255, 255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Store Related</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>220</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(132, 169, 140);
font: 14pt &quot;Nirmala UI&quot;;
color: rgb(255, 255, 255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Memberships</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>301</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid white;
background:none;
border-radius:10px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>350</y>
      <width>301</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid white;
background:none;
border-radius:10px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>370</y>
      <width>191</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="MemStatus">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(202, 210, 197);
color:rgb(0, 0, 0);
font: 350 13pt &quot;Nirmala UI&quot;;
border-radius:10px;
</string>
       </property>
       <property name="text">
        <string>Membership Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Rebate">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(202, 210, 197);
color:rgb(0, 0, 0);
font: 350 13pt &quot;Nirmala UI&quot;;
border-radius:10px;
</string>
       </property>
       <property name="text">
        <string>Membership Rebate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="MemPurchases">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(202, 210, 197);
color:rgb(0, 0, 0);
font: 350 13pt &quot;Nirmala UI&quot;;
border-radius:10px;
</string>
       </property>
       <property name="text">
        <string>Member Purchases</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addMembers">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(202, 210, 197);
color:rgb(0, 0, 0);
font: 350 13pt &quot;Nirmala UI&quot;;
border-radius:10px;
</string>
       </property>
       <property name="text">
        <string>Add/Delete Members</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>201</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="SalesReport">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(202, 210, 197);
color:rgb(0, 0, 0);
font: 350 13pt &quot;Nirmala UI&quot;;
border-radius:10px;
</string>
       </property>
       <property name="text">
        <string>Sales Report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StoreProducts">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(202, 210, 197);
color:rgb(0, 0, 0);
font: 350 13pt &quot;Nirmala UI&quot;;
border-radius:10px;
</string>
       </property>
       <property name="text">
        <string>Store Products</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SearchItem">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(202, 210, 197);
color:rgb(0, 0, 0);
font: 350 13pt &quot;Nirmala UI&quot;;
border-radius:10px;
</string>
       </property>
       <property name="text">
        <string>Search Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AddTrans">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(202, 210, 197);
color:rgb(0, 0, 0);
font: 350 13pt &quot;Nirmala UI&quot;;
border-radius:10px;
</string>
       </property>
       <property name="text">
        <string>Add a Transaction</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>widget</zorder>
   <zorder>label</zorder>
   <zorder>graphicsView</zorder>
   <zorder>addMembers</zorder>
   <zorder>AddTrans</zorder>
   <zorder>StoreProducts</zorder>
   <zorder>SalesReport</zorder>
   <zorder>Rebate</zorder>
   <zorder>SearchItem</zorder>
   <zorder>MemStatus</zorder>
   <zorder>MemPurchases</zorder>
   <zorder>DateTime</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
